### ISSUE TITLE
How to install OpenGapps using Gearlock in AndroidTV x82 ! 

### ISSUE THREAD

**Issue by Ares-03** [axonasif/gearlock#40]
Hello @axonasif , I'm currently running AndroidTV x86 32bite (Android version -8.1) on my old pc. I installed your GearLock in it to change the resolution. Now, i'm thinking of installing OpeenGapps in this by using Gearlock can you tell me how I can do that, i see in your Gearlock home page, you wrote you get success in installing OpeenGapps zips. So, can you tell me how I can do that in my case. Step by step..

I'm trying some methods on my own.. like :

First- I downloaded OpenGapps from the official website, i selected x86 on the platform, and 8.1 in Android, and tvstock in variants (because i want to install it on TV).

Second - I move this file to my tv os inaternal storage. Then i go in GearLock GUI by pressing alt F1 and then select Install/uninstall (GXPM) then install from custom location and choose Opengapp zip and install and then it installed then enter..

Is that the right method? Or tell your method?

**Comment by axonasif** [comment]
Hi @Ares-03, this is correct, did it not work?

**Comment by Ares-03** [comment]
> Hi @Ares-03, this is correct, did it not work?

No, I just thought of it, now I'll try it.  One more thing, I don't know why, but after booting Android TV x86 from pendrive.  After some time it gets stuck, 

Cpu - Pentium dual core E5500 (one thing about it, it does not have the support of so many things)
Ram - 2 GB
this is my old pc details, can you recommend any android os for my pc
